### PR TITLE
ARCHIE-1996 - Related document card

### PR DESCRIPTION
**Pull request by jaslakson**
* Related document card - might still need to add favorites, not sure yet
* A few linter errors for exhaustive dependencies

### CODE REVIEW

File: src/components/Cards/RelatedDocumentCard/RelatedDocumentCard.stories.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    `}
+  `,
+}
+
+const StoryWrapper = styled.div`
+  ${withThemes(StoryWrapperTheme)}
+`;
+
+export const Default = () => {
+  const [success, setSuccess] = useState(false);
```
Comment (mirdathebirda) [inline src/components/Cards/RelatedDocumentCard/RelatedDocumentCard.stories.js:38]: you can probably delete this - it doesn't look like this is used anywhere?

Comment (jaslakson) [reply src/components/Cards/RelatedDocumentCard/RelatedDocumentCard.stories.js:38]: ah yes, probably a copy pasta problem

File: src/components/Cards/RelatedDocumentCard/index.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    ${breakpoint('md')`
+      display: block;
+    `}`,
+};
+const RelatedDocumentCardSubtitle = styled.span.attrs({
+  className: 'related-document-card__subtitle',
+})`${withThemes(RelatedDocumentCardSubtitleTheme)}`;
+
+const StyledSticker = styled(Sticker)`
+`;
```
Comment (mirdathebirda) [inline src/components/Cards/RelatedDocumentCard/index.js:112]: if there are no additional styles being added to Sticker, you could delete these lines and use `<Sticker />` on line 48

**Review by ttornese — APPROVED** [review]

**Review by mirdathebirda — APPROVED** [review]

**Comment by atk-dm** [comment]
:rocket: PR was released in [`v1.5.0`](https://github.com/Americastestkitchen/mise-ui/releases/tag/v1.5.0) :rocket:

**Pull request merged**